host/xtest/pkcs11_1000.c

```diff
@@ -5458,3 +5458,628 @@ static void xtest_pkcs11_test_1019(ADBG_Case_t *c)
 }
 ADBG_CASE_DEFINE(pkcs11, 1019, xtest_pkcs11_test_1019,
 		 "PKCS11: Elliptic Curve key generation and signing");
+
+static void xtest_pkcs11_test_1020(ADBG_Case_t *c)
+{
+	CK_RV rv = CKR_GENERAL_ERROR;
+	CK_SLOT_ID slot = 0;
+	CK_SESSION_HANDLE session = CK_INVALID_HANDLE;
+	CK_FLAGS session_flags = CKF_SERIAL_SESSION | CKF_RW_SESSION;
+	CK_OBJECT_HANDLE wrapping_key1 = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE wrapping_key2 = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE wrapping_key_inv = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE key = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE key_sz24 = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE key_sens = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE key_inv = CK_INVALID_HANDLE;
+	CK_OBJECT_HANDLE unwrapped_key = CK_INVALID_HANDLE;
+	CK_ATTRIBUTE set_template[] = {
+		{ CKA_WRAP, &(CK_BBOOL){CK_TRUE}, sizeof(CK_BBOOL) },
```

Comment:
Add extra spaces after and before brackets:
`&(CK_BBOOL){ CK_TRUE }`

Apply same for other entries.

Reply:
Any specific reason for making this change ? Apart from test 1019, I see that all tests follow similar pattern i.e no extra spaces.

I can change it in this test case if you think it is required but then we should change it throughout the file

Reply:
I believe it was checkpatch that complained about it. I am happy either way.

Actually it complains with this now ;)
```
ERROR: space required before the open brace '{'
```

Reply:
I am not sure what the fix for that is :-)

I have seen these errors before but usually ignored them. @etienne-lms , what's the right way ?

Reply:
No strong opinion here. Since source file already has such _issues_, a global fix, from a specific P-R, could address that. As for this P-R, since checkpatch complains, maybe easier to address its comments :)

Reply:
The checkpatch warning gets fixed with the following 
&(CK_BOOL) {CK_TRUE};

i.e adding a space before "{". 

If you are fine with that, I will change it for this testcase atleast in this PR.

Reply:
Found it easier to fix throughout the file. Have added a patch on of the original patch to fix this.

Reply:
Note: for this P-R to pass checkpatch, i suggest you squash the fixup related to wrap test into commit "xtest: pkcs11: Add tests for wrap/unwrap operations" and have your second commit fixing the previously existing code.